### DISCUSSION TITLE
Final bundle size

### DISCUSSION THREAD

**Discussion by project0** [pmndrs/react-spring#1494] [Support]
I wonder if there is still room for optimization here? The final bundle size seems to be doubled since v8.

v 8.0.23
> dist/assets/vendor.9d333d0a.js       214.52kb / brotli: 60.25kb

v 9.1.2
> dist/assets/vendor.799cf757.js       230.98kb / brotli: 66.08kb

There was already some discussion around this, but it does not look like this has been solved:

https://github.com/pmndrs/react-spring/issues/836
https://github.com/pmndrs/react-spring/pull/632#issuecomment-538041865
https://github.com/pmndrs/react-spring/issues/1158

**Comment by joshuaellis** [comment]
There certainly is. Currently it's unsafe for us to specifically flag sideEffects:false in our package json and I haven't gotten round to investigating this.

If you're interested in contributing to solve this issue, feel free to make a PR.
